deploy/script/katello-backup

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+usage(){
+  echo "Usage: katello-backup <backup directory>"
+  echo " eg: $ katello-backup /tmp/katello-backup/"
+}
+
+if [ "$#" -ne 1 ]; then
+  echo "ERROR: Please specify an export directory"
+  usage
+  exit 1
+fi
+
+# Backup dir
+umask 0027
+export BDIR=$1
+mkdir $BDIR
+chgrp postgres $BDIR
+chmod 770 $BDIR
+cd $BDIR
+echo "Creating backup folder $BDIR"
+echo ""
+
+# Conf files
+echo "Backing up config files... "
+tar --selinux -czvf config_files.tar.gz \
+/etc/katello-installer \
+/etc/candlepin \
+/etc/foreman \
+/etc/foreman-proxy \
+/etc/gutterball \
+/etc/hammer \
+/etc/pulp \
+/etc/puppet \
+/etc/qpid \
+/etc/qpid-dispatch \
+/etc/elasticsearch \
+/etc/tomcat* \
+/etc/pki/katello \
+/etc/pki/katello-certs-tools \
+/etc/sysconfig/elasticsearch \
+/etc/sysconfig/tomcat* \
+/root/ssl-build \
+/var/www/html/pub
+echo "Done."
+echo ""
+
+echo "Backing up Elastic Search data... "
+tar --selinux -czvf elastic_data.tar.gz /var/lib/elasticsearch
+echo "Done."
+echo ""
+
+echo "Backing up Pulp data... "
+katello-service stop
+tar --selinux -cvf pulp_data.tar /var/lib/pulp/ /var/www/pub/
+katello-service start
+echo "Done."
+echo ""
+
+echo "Backing up postgres db... "
+tar --selinux -czvf pgsql_data.tar.gz /var/lib/pgsql/data/
+echo "Done."
+echo ""
+
+echo "Backing up mongo db... "
+tar --selinux -czvf mongo_data.tar.gz /var/lib/mongodb/
```

Comment:
we do want to backup `/var/lib/mongodb/`, be we need to exclude `/var/lib/mongodb/mongod.lock`. this is a pid file, and backing that up causes mongo to not start when restoring. This causes `katello-restore` to hang once it runs `katello-service restart`; mongo falls over, then celery sits (forever?) waiting to bring up nodes.